json/mainnet-bitcoin-config.json

```diff
@@ -1,14 +1,14 @@
 {
-  "bitcoinDataDirectory": "[FILL THIS IN!]",
```

Comment:
can we leave the `mainnet-bitcoin-config.json` and mainnet-core-config.json` and create separate config files just for the docker images? Because these `localhost` values are the default for running the node on a single machine without docker.

Reply:
ok, I create docker config files and submit again

Reply:
changed the config files.